src/lib.ts

```diff
@@ -1221,12 +1220,12 @@ export function setCombatFlags(
   return visitUrl(
     `account.php?${
       ([
-        ...flat(
-          flags.map(({ flag, value }) => [
+        flags
```

Comment:
`...` has gone missing, think it's still needed?

Also I think you can `flatMap` here?